app/frontend/wallet/shelley/helpers/version-check.ts

```diff
@@ -1,10 +1,5 @@
-interface Version {
-  major: string
-  minor: string
-  patch: string
-}
-
-export const hasRequiredVersion = (currentVersionStr: Version, requiredVersionStr) => {
+// TODO: type version
+export const hasRequiredVersion = (currentVersionStr, requiredVersionStr) => {
```

Comment:
Temporarily removed the typing since we should refactor this version checking and I don't want to address it in this PR (potentially making changes to all cryptoProviders and also actions, wallet etc.)